### ISSUE TITLE
Telegram Passport window not opening (multi-accounts)

### ISSUE THREAD

**Issue by leiniercs** [telegramdesktop/tdesktop#16155]
### Steps to reproduce
1. I select the account in the accounts list (I have 3)
2. I interact with my bot, from it I open the web page that shows me the 'Authorize' button (https://reputacionbot.ccari.net/autorizar_dev.html)
3. I press the 'Authorize' button in order to launch the Telegram Passport window on Telegram Desktop software.
4. The Telegram Passport window never opens

### Expected behaviour
Telegram Passport window should open based on the active account I have selected.

### Actual behaviour
The Telegram Passport window never opens.

### Configuration
**Operating system:**
Linux leiniercs 5.4.0-58-generic #64-Ubuntu SMP Wed Dec 9 08:16:25 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
**Version of Telegram Desktop:**
2.7.1
**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:
Official website
**Used theme**:
Dark
<details><summary><b>Logs</b>:</summary>
Insert log.txt here (if necessary)
</details>

**Comment by john-preston** [comment]
🤔  Works for me, two accounts, window opens:

**Comment by john-preston** [comment]
![image](https://user-images.githubusercontent.com/17900494/115196517-0e300780-a101-11eb-93dc-a692c3e8c8d9.png)

**Comment by john-preston** [comment]
Does the app open? Does it open on different t.me links? For example if you follow https://t.me/preston in the same browser?

**Comment by leiniercs** [comment]
Uhhmmmm. What happen then? I tried to send you a message from that URL and Telegram Desktop didn't opened the private chat with you. Maybe we found something different. I'm using Mozilla Firefox and I allowed it to open the 'tg://' links. What should be happening?

**Comment by leiniercs** [comment]
Good news. It is working now after I configured Mozilla Firefox again and reset the 'tg://' handler. Mozilla Firefox was the one causing this abnormal behavior. Closing the ticket. Thank you, Preston.

![image](https://user-images.githubusercontent.com/44554001/115257256-e5f2d980-a0fd-11eb-849f-4f18daa02da0.png)

![image](https://user-images.githubusercontent.com/44554001/115257189-d6739080-a0fd-11eb-9679-02b95b8e70a8.png)
